src/plugins/workspace/public/components/workspace_menu/workspace_menu.tsx

```diff
@@ -67,68 +86,68 @@ export const WorkspaceMenu = ({ coreStart }: Props) => {
     setPopover(false);
   };
 
-  const workspaceToItem = (workspace: WorkspaceObject) => {
+  const workspaceUseCaseToItem = (
+    workspace: WorkspaceObject,
+    index: number,
+    workspaceURL: string
+  ) => {
+    const useCase = workspace.features
+      ?.map(getUseCaseFromFeatureConfig)
+      .filter(isNotNull)
```

Comment:
Maybe we can simply do this:
```suggestion
      .filter(Boolean)
```